addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/handler/NestCameraHandler.java

```diff
@@ -0,0 +1,62 @@
+package org.openhab.binding.nest.handler;
+
+import static org.openhab.binding.nest.NestBindingConstants.*;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.library.types.StringType;
+import org.eclipse.smarthome.core.thing.Channel;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.nest.internal.data.Camera;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class NestCameraHandler extends BaseNestHandler {
```

Comment:
Please add a JavaDoc with `@author` tag containing a real name of the author. See [Code Style](http://docs.openhab.org/developers/development/guidelines.html#a-code-style).